pyproject.toml

```diff
@@ -109,6 +97,13 @@ dependencies = [
     "langchain-elasticsearch>=0.2.0",
 ]
 
+[tool.uv.sources]
+langflow-base = { path = "src/backend/base" }
+langflow = { workspace = true }
```

Comment:
Could probably remove this, since we're not utilizing workspaces

Reply:
Yeah..that's extremely annoying that debug doesn't work without workspaces (at least, with our current configuration).

Reply:
This should fix the vscode debug thing https://github.com/langflow-ai/langflow/pull/4072/commits/bd14fb50c8f095d5c975788378f1c2b0cf206f93 feel free to give it a try.

Reply:
Confirmed works for me, thanks